### ISSUE TITLE
CentOS/RHEL 6 - there's no python2-jinja2 available

### ISSUE THREAD

**Issue by bocekm** [AloisMahdal/jattool#3]
CentOS/RHEL 6 has python-jinja2 instead.
```
Error: Package: jattool-report-0.0.2-1.el6.noarch (amahdal-jats)
           Requires: python2-jinja2
```

**Comment by bocekm** [comment]
Workaround: I've found out there's a `jattool-minimal` in the copr repo, which doesn't require the `jinja`.

**Comment by AloisMahdal** [comment]
This is known and documented in README.  Machines where tests are to be only run should install jattool-minimal

> Also, older distros also necessarily have all dependencies available, so if you only want to run test (but not eg. export to HTML), you only need to install jattool-minimal

I don't see anythhing I can do about it now.

**Comment by bocekm** [comment]
You can update spec file for the rpm in similar fashion to this:
```
%package report
%if 0%{?rhel} && 0%{?rhel} < 7
Requires: python-jinja2
%else
Requires: python2-jinja2
%endif
```
I'm not in the office now. Some other day I'll check whether the syntax I pulled out my head is correct and send a pull request.

**Comment by AloisMahdal** [comment]
So you're saying there *is* jinja2 on RHEL6?  That changes everything! :-)

I'll apply the fix in the spec file then.

**Comment by AloisMahdal** [comment]
JFTR, turns out it's also python-jinja2 on RHEL-7, ie.

    %if 0%{?rhel} && 0%{?rhel} <= 7

(already fixed in v0.0.7)
